model/Export/AbstractQTIItemExporter.php

```diff
@@ -316,4 +326,26 @@ protected function getMetadataExporter()
         }
         return $this->metadataExporter;
     }
+
+    private function copySharedStimulusCssFiles(string $link, string $basePath): string
+    {
+        $asset = new core_kernel_classes_Resource(UriHelper::decode($link));
+        $baseName = 'assets/' . $this->getSharedStimulusCSSExporter()->getResourceLink($asset);
+        $this->getSharedStimulusCSSExporter()->pack(
+            $asset,
+            $this->getZip(),
+            $basePath . '/' . dirname($baseName)
```

Comment:
because i saw that we are not compatible with Windows directory separator i think this is a `good way` for the moment so the hardcoded '/' is the way to go for now